### DISCUSSION TITLE
ジャンパのハンダがつかなくなってしまいました。

### DISCUSSION THREAD

**Discussion by sayaryou** [Yowkees/keyball#158] [Q&A] [answered]
ジャンパのはんだをつけようとしていてなかなかつけることができず、ハンダが乗らなくなってしまいました。解決策はありますでしょうか。よろしくお願いいたします。
![EB8DCF67-6BCA-4A7B-B582-5CC633AF19F8](https://user-images.githubusercontent.com/118281171/201936567-54a324c8-59bc-4199-911e-47b226765bad.jpeg)

**Comment by rinyacho** [comment]
フラックスを使用されていない場合は、使ってみると良いと思います。
https://salicylic-acid3.hatenablog.com/entry/tool-lubrication

**Reply by sayaryou** [reply]
ご回答ありがとうございます。
フラックスも塗ってみましたが付かなかったです。
おそらく表面の銀色の部分が禿げてしまっているのだと思います。

**Reply by rinyacho** [reply, marked as answer]
回路図を見る限り、OLEDの回路の接続先はSDAかGNDだと思います（リバーシブルなので）。おそらくSDAですかね。
GNDの場合は、下記と同じような形で導線でショートさせる。
https://github.com/Yowkees/keyball/discussions/127
SDAの場合も、同じような形でOLEDの該当箇所とProMicroのSDAを導線でショートさせる。
ということかなぁと思いました。
参考程度に～。

**Reply by sayaryou** [reply]
情報ありがとうございます。
導線でGNDに繫ぐことで無事動作確認できました。

**Reply by rinyacho** [reply]
解決したみたいで良かったです。
GNDでしたか～。

**Comment by Yowkees** [comment]
フォローできていなくて申し訳ありませんでした。
Keyball39をお選びいただき誠にありがとうございます！

watobii様ご回答いただきありがとうございます！
無事に動作できたということで良かったです。
今後ともどうぞよろしくお願いいたします。
